domains/min.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "minhducle25",
+    "email": "min.qut.work@gmail.com"
+  },
+  "record": {
+    "CNAME": "51.79.135.245"
```

Comment:
Invalid, CNAME records must be valid hostnames not IPs. If you want to use an IP, use an A record.